libs/logger/formatters/workflowFormatter.php

```diff
@@ -8,40 +8,52 @@ class WorkflowFormatter {
     
     const TEMPLATES = [
         DataActions::ADD.'-'.LoggableTypes::WORKFLOW_STEP => [
-            "message"=>"Workflow Step %s - %s has been added",
+            "message"=>"Workflow Step <strong>%s</strong> - <strong>%s</strong> has been added",
             "variables"=>"stepID,stepTitle"
         ],
         DataActions::DELETE.'-'.LoggableTypes::WORKFLOW_STEP => [
-            "message"=>"Workflow Step %s deleted",
+            "message"=>"Workflow Step <strong>%s</strong> deleted",
             "variables"=>"stepID"
         ],
         DataActions::MODIFY.'-'.LoggableTypes::WORKFLOW_STEP => [
-            "message"=>"Workflow Step %s",
+            "message"=>"Workflow Step <strong>%s</strong>",
             "variables"=>"stepID,".FormatOptions::READ_COLUMN_NAMES,
             "loggableColumns"=>"posX,posY,stepTitle,indicatorID_for_assigned_empUID,indicatorID_for_assigned_groupID"
         ],
+        DataActions::ADD.'-'.LoggableTypes::EVENTS => [
+            "message"=>"Event <strong>%s</strong> has been added",
+            "variables"=>"eventDescription"
+        ],
+        DataActions::DELETE.'-'.LoggableTypes::EVENTS => [
+            "message"=>"Event <strong>%s</strong> has been deleted",
+            "variables"=>"eventID"
+        ],
+        DataActions::MODIFY.'-'.LoggableTypes::EVENTS => [
+            "message"=>"Event <strong>%s</strong> has been modified",
+            "variables"=>"eventDescription"
+        ],
         DataActions::ADD.'-'.LoggableTypes::ROUTE_EVENTS => [
-            "message"=>"Workflow %s Step %s %s action has been added",
+            "message"=>"Workflow <strong>%s</strong> Step <strong>%s</strong> <strong>%s</strong> action has been added",

```

Comment:
Looks like "They" changed their mind on <strong>  That was supposed to be deprecated.  Anyway, we probably don't want to use it _unless_ we define an over-ride in our css.  Otherwise, it leaves it up to different browsers to interpret  it differently.

Reply:
Yeah this is the well known logger that we need to revamp, I was just making the key vars bolded. Should I change it back? It doesn't follow a css.